src/wovnio/wovnphp/Store.php

```diff
@@ -48,6 +43,20 @@ public function __construct($userSettings)
         }
     }
 
+    /**
+     *  Configuration validate
+     *
+     *  @return boolean
+     */
+    public function isValid()
+    {
+        return (
+            $this->settings['project_token']
+            && $this->settings['default_lang']
+            && $this->settings['supported_langs']
```

Comment:
I think `url_pattern` also required.

Reply:
The default value of `url_pattern_name` is `query`, so it is not necessary to check it.

Reply:
I agree, we usually expect customers to specify url_pattern themselves

Reply:
Oh, but you should check `url_pattern_name`, it's required.